app/src/main/java/com/jedsada/dangerplayground/UserModel.kt

```diff
@@ -1,6 +1,6 @@
 package com.jedsada.dangerplayground
 
-data class UserDao (val username : String? = null,
+data class UserModel (val username : String? = null,
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/T-Jedsada/danger-playground/blob/91c47854f91b127a896e7ea792d87395e683df47/app/src/main/java/com/jedsada/dangerplayground/UserModel.kt#L3"></span>Parameter should be on a separate line (unless all parameters can fit a single line)</td>
    </tr>
  </tbody>
</table>